requirements/base.txt

```diff
@@ -18,14 +18,15 @@ django-storages==1.9.1    # via -r requirements/base.in
 django-taggit==0.23.0     # via wagtail
 django-treebeard==4.3.1   # via wagtail
 django-waffle==0.20.0     # via -r requirements/base.in, edx-django-utils, edx-drf-extensions
-django==1.11.29           # via -r requirements/base.in, django-cors-headers, django-storages, django-taggit, django-treebeard, djangorestframework, edx-auth-backends, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition, wagtail
-djangorestframework-jwt==1.11.0  # via -r requirements/base.in, edx-drf-extensions
-djangorestframework==3.11.0  # via -r requirements/base.in, django-rest-swagger, edx-drf-extensions, rest-condition, wagtail
+django==1.11.29           # via -r requirements/base.in, django-cors-headers, django-storages, django-taggit, django-treebeard, drf-jwt, edx-auth-backends, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition, wagtail
+djangorestframework-jwt==1.11.0  # via -r requirements/base.in
```

Comment:
With `edx-drf-extensions` having switched to `drf-jwt`, can we drop this dependency now?  They use the same package name for imports, so we shouldn't have both installed.

Reply:
done.